### DISCUSSION TITLE
SAE kein Zugriff über Webbrowser möglich

### DISCUSSION THREAD

**Discussion by mgdrums** [camueller/SmartApplianceEnabler#279] [Installation und Betrieb]
Hallo, 

ich bin verzweifelt auf der Suche was ich falsch mache und muss dazu sagen, ich habe nicht so viel Ahnung von Linux. 
Ich habe die Installation in einem Docker Container wie in der Anleitung beschrieben durchgeführt. Die Anwendung ist auch active (siehe Screenshot). Nur leider kann ich über den Webbrowser und Port 8080 nicht darauf zugreifen. 
Hat jemand eine Idee was ich falsch mache? 
Vielen Dank und viele Grüße,
Michael
![docker version](https://user-images.githubusercontent.com/101500199/172821635-d31d86f6-c83a-445e-a522-0f7d6c3ea311.png)
![wireshark](https://user-images.githubusercontent.com/101500199/172821640-c6cc7352-e5f8-47d3-9d1a-b9ac4d4f208e.png)
![docker sae](https://user-images.githubusercontent.com/101500199/172821642-998bb3ed-671e-4e6f-9f33-3a6c81a2ab10.png)

**Comment by camueller** [comment]
Du schreibst, dass Du nicht viel Ahnung von Linux hast. Warum verwendest Du dann Docker?? Ohne ausreichend Linux-Erfahrung wirst Du so nicht glücklich werden. Warum verwendest Du nicht die Standard-Installation ohne Docker? Die ist genau für User mit wenig Linux-Erfahrung gedacht.

Ich muss Dir ehrlich sagen, dass mir meine Zeit zu schade dafür ist, in solchen Fällen zu supporten. Vielleicht haben andere User mehr Lust dazu, Dir weiterzuhelfen ...

**Reply by mgdrums** [reply]
Danke für deine schnelle Antwort. 
Da der Raspberry Pi momentan überall ausverkauft ist habe ich mir eine Alternative (Rock PI) besorgt. 
Somit funktioniert die Standard Installation nicht. Dachte mir, ich installiere Ubuntu und gehe den Weg über das Docker Image. 
Kann das überhaupt funktionieren?

**Reply by camueller** [reply]
Wenn man sich mit Linux auskennt: ja (wenn man keine Zugriffe auf GPIOs benötigt)
